che-editors.yaml

```diff
@@ -77,6 +77,8 @@ editors:
         volumeMounts:
           - name: plugins
             path: "/plugins"
+          - name: home
```

Comment:
I am not sure `home` is the right name to give. Users may use that name for a volume for the user home (`/home/theia` or `/home/user`). I would name it `theia-local` or something like that.

Reply:
`theia-local-config`

Reply:
fixed

Reply:
or `theia-local-storage`

Reply:
I would keep `theia-local` if there is no objections?

Reply:
@vinokurig I can see only one of the 2 files updated

Reply:
@sunix updated the secnd one, thanks.